docs/decision-maker-transaction.md

```diff
@@ -32,104 +32,106 @@ INPUT_FILE = "input.txt"
+    # Set up a wallet
+    wallet = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE_1})
 
-``` python
-    # Set up the wallets
-    wallet_1 = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE_1})
-    wallet_2 = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE_2})
+    # Set up an identity
+    identity = Identity(
+        name="my_aea", addresses=wallet.addresses, default_address_key=FETCHAI,
+    )
 ```
 
 ## Generate wealth
 
-Since we want to send funds from `wallet_1` to `wallet_2`, we need to generate some wealth for the `wallet_1`. We can
-do this with the following code
+Since we want to send funds from our AEA's `wallet`, we need to generate some wealth for the `wallet`. We can
+do this with the following code where we use the default address
+
 ``` python
-    # Generate some wealth
-    _try_generate_testnet_wealth(FETCHAI, wallet_1.addresses[FETCHAI])
+    # Generate some wealth for the default address
+    _try_generate_testnet_wealth(FETCHAI, identity.address)
 ```
 
-
 ## Create LedgerApis
 
 We need to create the LedgerApis object to be able to interact with the Fetch.ai `testnet`
+
 ``` python
-    # Set up the LedgerApis
+    # Set up the ledger apis for Fetch.AI testnet
     ledger_apis = LedgerApis({FETCHAI: {"network": "testnet"}}, FETCHAI)
 ```
 
 ## Create the aea
 
-To have access to the decision-maker, we need to create an AEA. An AEA constructor needs some variables to be passed `AEA(Identity, Connection, Wallet, LedgerApis, Resources)`
+To have access to the decision-maker, which is responsible for signing transactions, we need to create an AEA. An AEA constructor needs some variables to be passed `AEA(Identity, Connection, Wallet, LedgerApis, Resources)`
 So let's create these dependencies before we instantiate the AEA.
 
 ``` python
-    tx_handler = TransactionHandler(skill_context="skill_context", name="fake_skill")
+    # Initialize an empty set of resources
     resources = Resources()
-    resources.handler_registry.register(
-        (
-            PublicId.from_str("fetchai/fake_skill:0.1.0"),
-            PublicId.from_str("fetchai/internal:0.1.0"),
-        ),
-        tx_handler,
-    )
 
+    # Use the default stub connection
     stub_connection = StubConnection(
         input_file_path=INPUT_FILE, output_file_path=OUTPUT_FILE
     )
 ```
 
-We need to create an identity for our AEA and the counterparty since the only one who must have access to the wallet is the decision-maker.
+Create the `AEA` next.
 
 ``` python
-    identity_1 = Identity(
-        name="my_aea",
-        address=wallet_1.addresses.get(FETCHAI),
-        default_address_key=FETCHAI,
-    )
+    # create the AEA
+    my_aea = AEA(identity, [stub_connection], wallet, ledger_apis, resources)
+```
 
-    identity_2 = Identity(
-        name="my_aea_2",
-        address=wallet_2.addresses.get(FETCHAI),
-        default_address_key=FETCHAI,
+Add a simple skill with a transaction handler.
+
+``` python
+    skill_context = SkillContext(my_aea.context)
```

Comment:
example of programmatically creating a skill. still quite verbose but at least possible with the public api now.